docs/pages/blog.tsx

```diff
@@ -177,24 +209,15 @@ export default function Blog(props: InferGetStaticPropsType<typeof getStaticProp
             }}
           >
             {[firstPost, secondPost].map((post) => (
-              <Box
+              <Paper
+                component={Link}
                 key={post.slug}
-                sx={{
-                  display: 'flex',
-                  flexDirection: 'column',
-                  alignItems: 'initial',
-                  p: 2,
-                  border: 1,
-                  borderRadius: 1,
-                  borderColor: (theme) =>
-                    theme.palette.mode === 'dark'
-                      ? theme.palette.primaryDark[700]
-                      : theme.palette.grey[200],
-                  background: (theme) =>
-                    theme.palette.mode === 'dark'
-                      ? `linear-gradient(180deg, ${theme.palette.primaryDark[900]} 0%, #001E3C 100%)`
-                      : '#fff',
-                }}
+                href="/blog/"
```

Comment:
@siriwatknp I left the `href` like this only so the deploy preview doesn't break but my intention was to have the whole Paper clickable. I tried using `/blog/${props.slug}` but for some reason, it didn't work, so giving you a heads up :)